vc_zoom/indico_vc_zoom/api/client.py

```diff
@@ -0,0 +1,243 @@
+        )
+
+    def create(self, user_id, **kwargs):
+        if kwargs.get("start_time"):
+            kwargs["start_time"] = format_iso_dt(kwargs["start_time"])
+        return self.session.post(
+            "{}/users/{}/webinars".format(self.base_uri, user_id),
+            json=kwargs
+        )
+
+    def get(self, meeting_id, **kwargs):
+        return self.session.get("{}/webinars/{}".format(self.base_uri, meeting_id), json=kwargs)
+
+    def update(self, meeting_id, **kwargs):
+        if kwargs.get("start_time"):
+            kwargs["start_time"] = format_iso_dt(kwargs["start_time"])
+        return self.session.patch(
+            "{}/webinars/{}".format(self.base_uri, meeting_id), json=kwargs
+        )
+
+    def delete(self, meeting_id, **kwargs):
+        return self.session.delete(
+            "{}/webinars/{}".format(self.base_uri, meeting_id), json=kwargs
+        )
+
+
+class UserComponent(BaseComponent):
+    def me(self):
+        return self.get('me')
+
+    def list(self, **kwargs):
+        return self.session.get("{}/users".format(self.base_uri), params=kwargs)
+
+    def create(self, **kwargs):
+        return self.session.post("{}/users".format(self.base_uri), params=kwargs)
+
+    def update(self, user_id, **kwargs):
+        return self.session.patch("{}/users/{}".format(self.base_uri, user_id), params=kwargs)
+
+    def delete(self, user_id, **kwargs):
+        return self.session.delete("{}/users/{}".format(self.base_uri, user_id), params=kwargs)
+
+    def add_assistant(self, user_id, **kwargs):
+        return self.session.post("{}/users/{}/assistants".format(self.base_uri, user_id), json=kwargs)
+
+    def get_assistants(self, user_id, **kwargs):
+        return self.session.get("{}/users/{}/assistants".format(self.base_uri, user_id), params=kwargs)
+
+    def get(self, user_id, **kwargs):
+        return self.session.get("{}/users/{}".format(self.base_uri, user_id), params=kwargs)
+
+
+class ZoomClient(object):
+    """Zoom REST API Python Client."""
+
+    _components = {
+        'user': UserComponent,
+        'meeting': MeetingComponent,
+        'webinar': WebinarComponent
+    }
+
+    def __init__(self, api_key, api_secret, timeout=15):
+        """Create a new Zoom client.
+
+        :param api_key: the Zoom JWT API key
+        :param api_secret: the Zoom JWT API Secret
+        :param timeout: the time out to use for API requests
+        """
+        BASE_URI = "https://api.zoom.us/v2"
+
+        # Setup the config details
+        config = {
+            "api_key": api_key,
+            "api_secret": api_secret
+        }
+
+        # Instantiate the components
+
+        self.components = {
+            key: component(base_uri=BASE_URI, config=config, timeout=timeout)
+            for key, component in self._components.viewitems()
+        }
+
+    @property
+    def meeting(self):
+        """Get the meeting component."""
+        return self.components.get("meeting")
```

Comment:
```suggestion
        return self.components['meeting']
```

since we always have those components